### PR TITLE
Help Center: Add support for WP Support 3 tab component

### PR DESCRIPTION
**Pull request by agrullon95**
Fixes: https://github.com/Automattic/dotcom-forge/issues/9818

## Proposed Changes

* Add support for WPSupport 3 Tab component

CleanShot 2024-12-02 at 13 48 55@2x

## Why are these changes being made?

* WPSupport3 tab component is currently hidden. These changes add its functionality and make it appear on support docs  within Help Center.

## Testing Instructions

* Checkout trunk or use live link
* Navigate to `home?flags=help-center-experience` and open the Help Center
* Find a support doc that uses the wpsupport3 tab component (Example articles to search for: `Upload a theme`, `Get more storage space`, `Set the homepage`.
* Test its functionality
* Bonus: Find more support docs with this components and try to break it.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: packages/help-center/src/components/help-center-article.tsx [hunk 1]
```diff
@@ -53,6 +55,66 @@ export const HelpCenterArticle = () => {
 		}
 	}, [ post, query, sectionName ] );
 
+	const toggleTab = ( element: Element, show: boolean ) => {
```
Comment (escapemanuele) [inline packages/help-center/src/components/help-center-article.tsx]: Would it be possible to extract the tabbing functionality in a separate hook that we can call here?

Comment (agrullon95) [reply packages/help-center/src/components/help-center-article.tsx]: 🤔 Let me check how I can refactor this

**Review by escapemanuele — APPROVED** [review]
Woah, looks great now! Thanks Anthony

File: packages/help-center/src/components/help-center-article.scss [hunk 2]
```diff
@@ -455,10 +455,70 @@
...
+					white-space: nowrap;
+					overflow-x: auto;
+					border-bottom: 1px solid var( --wp--preset--color--border-light-gray, #eee );
+
+					.wpsupport3-tab__title {
+						padding: 8px 0;
+			
+						color: var( --blue-blue-50, #0675c4 );
+						/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+						font-size: 16px;
```
Comment (escapemanuele) [inline packages/help-center/src/components/help-center-article.scss]: I believe we don't need the 
```
/* stylelint-disable-next-line declaration-property-unit-allowed-list */
```
And we can use $font-body;

**Pull request merged**